src/components/Application/ProgressBarNEW.tsx

```diff
@@ -98,7 +127,7 @@ const ProgressBarNEW: React.FC<ProgressBarProps> = ({ structure, requestRevalida
         children: (
           <Grid>
             <Grid.Column width={4} textAlign="right" verticalAlign="middle">
-              {progress && getIndicator(progress, stepNumber)}
+              {progress && getIndicator(progress, isIsStrictSection, stepNumber)}
```

Comment:
```suggestion
              {progress && getIndicator(progress, isStrictSection, stepNumber)}
```

Reply:
I did one global replace to make sure we capture all of the places this was mistyped: c7093b4